tools/segments/config-dev.json

```diff
@@ -0,0 +1,80 @@
+{
+  "total": 18,
+  "offset": 0,
+  "limit": 18,
+  "data": [
+    {
+      "key": "commerce.headers.cs.Magento-Customer-Group",
+      "value": "b6589fc6ab0dc82cf12099d1c2d40ab994e8410c"
+    },
+    {
+      "key": "commerce.headers.cs.Magento-Environment-Id",
+      "value": "0a9ae65d-c1a5-49f6-af64-1ec1ff47ab52"
+    },
+    {
+      "key": "commerce.headers.cs.Magento-Store-Code",
+      "value": "main_website_store"
+    },
+    {
+      "key": "commerce.headers.cs.Magento-Store-View-Code",
+      "value": "default"
+    },
+    {
+      "key": "commerce.headers.cs.Magento-Website-Code",
+      "value": "base"
+    },
+    {
+      "key": "commerce.headers.cs.x-api-key",
+      "value": "9753cd30401a477e816ed850c4f77e18"
+    },
+    {
+      "key": "commerce-base-currency-code",
+      "value": "USD"
+    },
+    {
+      "key": "commerce-core-endpoint",
+      "value": "http://adobecommerce247.local/graphql"
+    },
+    {
+      "key": "commerce-endpoint",
+      "value": "http://adobecommerce247.local/graphql"
```

Comment:
Is this specific to your own setup? Are there instructions for how to update/change this?

Reply:
Hi @sirugh, thanks for your feedback - yes, that is my local instance I was using during development, I will restore the original one.